keps/sig-auth/541-external-credential-providers/kep.yaml

```diff
@@ -22,4 +22,4 @@ latest-milestone: "v1.20"
 milestone:
   alpha: "v1.10"
   beta: "v1.11"
-  stable: "v1.21"
+  stable: "v1.22"
```

Comment:
@annajung - continuing conversation[0] about updating this kep for 1.21, I updating this stable target per your note, let me know if this is not what we need.

[0] https://kubernetes.slack.com/archives/C0EN96KUY/p1611762222015600?thread_ts=1611606235.003600&cid=C0EN96KUY

Reply:
This looks good. In addition to this, I would ask to change the `latest-milestone` value to `v.1.21`.

Reply:
sounds good! thanks for taking the time to review. I gave this a go here: 9a8390fbb8cb7744aa7fd05d0b2d648ac89eb94a